jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -598,3 +598,11 @@
         frequency: 'H H/12 * * *'
         trigger-job: 'kubernetes-build'
     # END KUBEMARK
+    - fejta-soak-deploy:
+        job-name: ci-fejta-soak-deploy
+        frequency: '@daily'
+        trigger-job: 'kubernetes-build-1.2'
```

Comment:
mostly my suggestion is to be more explicit that you're avoiding triggering, rather than implicitly avoiding triggering.